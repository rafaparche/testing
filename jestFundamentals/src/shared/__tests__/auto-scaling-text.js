/**
 * @jest-environment jsdom
 */


import React from 'react'
import AutoScalingText from '../auto-scaling-text'
import {render} from '@testing-library/react'


test('renders', () => {
  render(<AutoScalingText />)
})

test('mounts', () => {
  const {container} = render(<AutoScalingText value="0" />)
  expect(container.firstChild).toMatchSnapshot()
})
