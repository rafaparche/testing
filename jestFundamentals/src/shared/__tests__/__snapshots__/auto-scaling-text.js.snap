// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mounts 1`] = `
<div
  class="autoScalingText"
  style="transform: scale(1,1);"
/>
`;
